dot/types/body.go

```diff
@@ -18,14 +18,13 @@ package types
 
 import (
 	"bytes"
-	"errors"
 	"fmt"
-	"io"
 	"math/big"
 
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/common/optional"
 	"github.com/ChainSafe/gossamer/lib/scale"
```

Comment:
we are using the original scale here, did you want to change it to only use scalev2 or do that in a follow up?

Reply:
My thought process was given the size of the PR, I would just change what I needed to in order to get everything stable with respect to the new digest changes, so I was thinking I would do that in a follow up. Of course I can add it to this PR if you see fit